pkg/cli/formatter/yaml.go

```diff
@@ -37,5 +38,10 @@ func (f *YamlFormatter) Format(resp *http.Response) ([]byte, error) {
 		return nil, err
 	}
 
+	// Response status is not 200.
```

Comment:
How about compare status code before Unmarshal  body?

Reply:
Unmarshal body before comparing the status is used for output the body while status is not 200.